docs/platforms/android/troubleshooting/shared-environments/index.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Shared Environments
+description: "Learn how to use the Sentry SDK within another SDK."
+sidebar_order: 1
+---
+
+<Note>
+
+Using the sentry SDK within another SDK is discouraged. This can lead to unexpected behavior and potential data leakage. If you need to use Sentry within another SDK, please follow the best practices outlined below.
```

Comment:
```suggestion
Using the Sentry SDK within another SDK is [discouraged](https://docs.sentry.io/platforms/). This can lead to unexpected behaviour and potential data leakage. If you need to use Sentry within another SDK, please follow the best practices outlined below.
```